vyper/builtin_functions/functions.py

```diff
@@ -1607,7 +1572,7 @@ def build_LLL(self, expr, context):
             [
                 "if",
                 ["slt", "orig", 0],
-                # CMC 2022-02-05 pretty sure this assertion is pointless.
+                # check orig != -2**255 (0 - (-2**255) == -2**255)
```

Comment:
Does this comment need to be fixed? What does it mean?

Reply:
i double checked and my original comment doesn't apply because the runtime assertion is required.

in twos-complement arithmetic, -1 * 0x80..00 == 0x80..00. on the EVM, this means `abs(-2**255) == -2**255`, which is not good, so we need a runtime clamp to make sure that the result is non-negative.